edge/pkg/clusterd/mission_deployer.go

```diff
@@ -189,6 +242,22 @@ func buildMissionYaml(input *edgeclustersv1.Mission) (string, error) {
 		input.Spec.StateCheck.Command = " "
 	}
 
+	if input.Spec.MissionResource == "" {
+		input.Spec.MissionResource = " "
```

Comment:
curious, why " " instead of ""?

Reply:
I have a comment above to explain it:
	// probably due to the json encoder in arktos, the commmnd "kubectl apply mission" in arktos
	// fails if the mission.StateCheck.Command is nil or empty.
	// We trick it with a string with one space.